src/globals/scss/overrides/_spacing.scss

```diff
@@ -14,25 +14,30 @@ $spacing-directions: (
 // .govuk-\!-m-0 {
 //   margin: 0;
 // }
-// 
+//
 // [..]
-// 
+//
 // .govuk-\!-mt-1 {
 //   margin-top: [whatever spacing point 1 is...]
 // }
 
 @mixin generate-spacing-overrides($property, $property-shorthand) {
   // For each point in the spacing scale (defined in settings), create an
   // override that affects all directions...
-  @each $scale-point, $scale-value in $govuk-spacing-scale {
+  @each $scale-point, $scale-map in $govuk-spacing-responsive-scale {
+
     .govuk-\!-#{$property-shorthand}-#{$scale-point} {
```

Comment:
We should probably have something in the class names to mark these as responsive overrides?

Reply:
As just discussed, we now have an "r" to indicate responsiveness.